### ISSUE TITLE
A11y: No focus indicator when more actions button has focus

### ISSUE THREAD

**Issue by jxjj** [UMN-LATIS/z#200]
![ScreenShot 2024-12-26 at 13 17 57@2x](https://github.com/user-attachments/assets/7e6abd15-fe71-4cd6-9ed1-9ee95e61dba8)
